CHANGELOG.rst

```diff
@@ -170,7 +170,7 @@ Version 10.15.1
 * `@stsewd <https://github.com/stsewd>`__: Update docs requirements (`#11032 <https://github.com/readthedocs/readthedocs.org/pull/11032>`__)
 * `@github-actions[bot] <https://github.com/github-actions[bot]>`__: Dependencies: all packages updated via pip-tools (`#11029 <https://github.com/readthedocs/readthedocs.org/pull/11029>`__)
 * `@stsewd <https://github.com/stsewd>`__: Redirects: limit to 100 per project (`#11028 <https://github.com/readthedocs/readthedocs.org/pull/11028>`__)
-* `@humitos <https://github.com/humitos>`__: Build: reset notifications when reseting a build (`#11027 <https://github.com/readthedocs/readthedocs.org/pull/11027>`__)
+* `@humitos <https://github.com/humitos>`__: Build: reset notifications when resetting a build (`#11027 <https://github.com/readthedocs/readthedocs.org/pull/11027>`__)
```

Comment:
I think there is no need to touch this file. Also, changing the name of the PRs here won't change them in GH and these won't match.

Reply:
my 1c: IMHO it is worth fixing it as it is "user facing" , whenever PRs with their titles -- developer facing etc.

But if you insist - I can easily skip it. Just one more call ;)